src/operator/expand-support.ts

```diff
@@ -46,19 +53,26 @@ export class ExpandSubscriber<T, R> extends OuterSubscriber<T, R> {
       let result = tryCatch(this.project)(value, index);
       if (result === errorObject) {
         destination.error(result.e);
+      } else if (!this.scheduler) {
+        this.subscribeToProjection(result, value, index);
```

Comment:
Looks like we've lost the scalar optimization for this branch of code. `if (result._isScalar) {` ...etc.